components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -533,6 +577,8 @@ private void updateInboundAuthRequestConfiguration(int applicationId,
             }
 
             inboundAuthReqConfigPrepStmt.executeBatch();
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
This should be fixed. This method throws IdentityApplicationManagementException. So a IdentityApplicationManagementException should be thrown. For the moment replaced with log.error() at #440